### DISCUSSION TITLE
How to recording while streaming?

### DISCUSSION THREAD

**Discussion by drequeary** [streamlink/streamlink#4450] [Q&A] [answered]
How do I get streamlink to open my player and record stream? I use a command similar to this 

`streamlink https://www.youtube.com/watch?v=XDh0JcxrbPM best --output file.mp4` 

but that just records. I use PotPlayer on Windows btw. Or is this not possible with the windows version?

**Comment by mkbloke** [comment, marked as answer]
https://streamlink.github.io/cli.html#cmdoption-r

**Reply by drequeary** [reply]
Ah thanks
